test/test-model.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+var assert = require('yeoman-assert');
+var helpers = require('yeoman-test');
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+
+
+describe('generator-alfresco:model', function () {
+  this.timeout(60000);
+  var osTempDir = path.join(os.tmpdir(), 'temp-test');
+
+  before(function (done) {
+    helpers.run(path.join(__dirname, '../generators/app'))
+      .inDir(osTempDir)
+      .withOptions({ 'skip-install': true })
+      .withPrompts({
+        sdkVersion: '2.1.1',
+        projectArtifactId: 'temp-test',
+      })
+      .on('end', done);
+  });
+
+  it('create with no options and prompts for coverage', function(done) {
```

Comment:
I wonder if we could remove this since we externalized and added tests for your filter functions? May require a bit more coverage the other code, but will likely be much cleaner than this. It's always a bit yucky to have a test that doesn't actually assert anything :)